app/util/git_client_check.py

```diff
@@ -0,0 +1,7 @@
+from subprocess import check_output
+
+
+try:
+    print("Git version: {}".format(check_output(["git", "--version"])))
+except Exception:
+    raise Exception("Please check if git installed and available from command line.")
```

Comment:
Do we really support all git versions? Is `git clone --no-checkout` supported from the first git version?

Reply:
`git` version 1.8.2 has `--no-checkout` flag, checked

Reply:
Good question.
@SergeyMoroz0703+++++